### DISCUSSION TITLE
Switching from private to public for contact info

### DISCUSSION THREAD

**Discussion by DerekDen** [code4recovery/12-step-meeting-list#925] [Q&A]
Hi, I have just tried to switch our site to have public contact info (was previously private), and no change occurs on the listings.  Have I missed some other change I need to do?.

Our site is aaportugal.net

<img width="420" alt="Screenshot 2022-09-16 at 11 00 45" src="https://user-images.githubusercontent.com/112857352/190614199-9087703c-6296-40b4-a161-e2409f064b66.png">
<img width="946" alt="Screenshot 2022-09-16 at 11 00 04" src="https://user-images.githubusercontent.com/112857352/190614206-b2edd995-ab28-4cca-8ce4-89220e76c924.png">
<img width="383" alt="Screenshot 2022-09-16 at 10 59 17" src="https://user-images.githubusercontent.com/112857352/190614207-46cd9f18-de4d-41ed-ab6a-bc1d6f1f4cd5.png">

in fellowship, Derek

**Comment by DerekDen** [comment]
I have purged all the caches server and client side that I can....

**Comment by joshreisner** [comment]
Hi @DerekDen - support in TSML UI is coming soon, probably within the next week or so. https://github.com/code4recovery/tsml-ui/issues/253

For the time being, if you prefer, you can switch to the "Legacy UI"

**Reply by DerekDen** [reply]
Thanks Josh, but I was already using Legacy UI (I had switched a few times to see what the differences were).

**Comment by DerekDen** [comment]
It's just started working....  I'm not sure what was going on.  I wish I weren't new to wordpress and web development (I was a C programmer prior to retiring).

**Reply by DerekDen** [reply]
I thinking I might just be too impatient.

**Reply by joshreisner** [reply]
I think now you are seeing the `email` field working, which is supported by TSML UI. What you're not seeing still are the individual contacts, like you and Anneli in your screenshot above

**Reply by DerekDen** [reply]
I am using the Legacy UI, and the Public setting for contacts.  Using the list as an end user would, I don't see the contacts, HOWEVER if I click the "view" link in the meeting list editor I do.  I have been retired too long ;-)

**Reply by joshreisner** [reply]
I think what has happened is your main `/meetings` page has been customized to redirect to `/aa-meetings`, where TSML UI is enabled. If that redirect were not in place you'd be seeing the legacy interface there. I think that is causing some bizarre behavior, maybe remove the redirect and the `/aa-meetings` page?

**Reply by DerekDen** [reply]
Thanks, I'm getting out of my depth here, I'll contact the original developer
